code/src/components/TaskItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import tasks from 'reducers/tasks';
+
+const TaskItem = ({ itemIndex }) => {
+	const item = useSelector((store) => store.tasks.list.items[itemIndex]);
+	const dispatch = useDispatch();
+
+	const handleCheckbox = () => {
+		dispatch(
+			tasks.actions.toggleDone({
+				itemIndex: itemIndex,
+				isComplete: !item.isComplete,
+			})
+		);
+	};
+
+	const handleDelete = () => {
+		dispatch(
+			tasks.actions.deleteTask({
+				itemIndex: itemIndex,
+			})
+		);
+	};
+
+	return (
+		<div id='task-list' className={`${item.isComplete ? 'complete' : ''}`}>
```

Comment:
love the styling for completed tasks with the overlined text and nice solution with ternary operator